openapi/data_repository_service_OAS3.0.swagger.yaml

```diff
@@ -0,0 +1,1220 @@
+openapi: 3.0.0
```

Comment:
Does this replace `openapi/data_repository_service.openapi.yaml`?

Reply:
Natan has code to generate this so we can delete

Reply:
OK, deleted this file since you have code on master to generate v3.0 swagger yaml